ts/features/wallet/onboarding/privative/screens/add/AddPrivativeCardComponent.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import { SafeAreaView, StyleSheet } from "react-native";
+import { View } from "native-base";
+import { ScrollView } from "react-native-gesture-handler";
+import I18n from "../../../../../../i18n";
+import { PaymentInstrument } from "../../../../../../../definitions/pagopa/walletv2/PaymentInstrument";
+import BaseScreenComponent from "../../../../../../components/screens/BaseScreenComponent";
+import { IOStyles } from "../../../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../../../components/core/typography/H1";
+import {
+  getPrivativeGdoLogoUrl,
+  getPrivativeLoyaltyLogoUrl,
+  getTitleFromPaymentInstrument,
+  isCoBadgeOrPrivativeBlocked
+} from "../../../../../../utils/paymentMethod";
+import { InfoBox } from "../../../../../../components/box/InfoBox";
+import { IOColors } from "../../../../../../components/core/variables/IOColors";
+import { Body } from "../../../../../../components/core/typography/Body";
+import FooterWithButtons from "../../../../../../components/ui/FooterWithButtons";
+import {
+  cancelButtonProps,
+  confirmButtonProps
+} from "../../../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import BasePrivativeCard from "../../../../privative/component/card/BasePrivativeCard";
+
+type Props = {
+  pan: PaymentInstrument;
+  handleContinue: () => void;
+  handleSkip: () => void;
+} & Pick<React.ComponentProps<typeof BaseScreenComponent>, "contextualHelp">;
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: "center"
+  },
+  title: { lineHeight: 33, alignSelf: "flex-start" },
+  flexStart: { alignSelf: "flex-start" }
+});
+
+const loadLocales = () => ({
+  headerTitle: I18n.t("wallet.onboarding.privative.headerTitle"),
+  screenTitle: I18n.t("wallet.onboarding.privative.add.screenTitle"),
+  blockedCard: I18n.t("wallet.onboarding.privative.add.blocked")
+});
+
+const AddPrivativeCardComponent: React.FunctionComponent<Props> = (
+  props: Props
+) => {
+  const { headerTitle, screenTitle, blockedCard } = loadLocales();
+
+  const caption = getTitleFromPaymentInstrument(props.pan);
```

Comment:
Is not strictly connected with this pr, but what you think about change the behaviour of `getTitleFromPaymentInstrument`?
With the current implementation, in case we receive an undefined `panPartialNumber`, we show:
![Schermata 2021-03-18 alle 12 07 21](https://user-images.githubusercontent.com/26501317/111616922-c278fe00-87e2-11eb-8129-4491b4c3bbf3.png)

My proposal:
```typescript
export const getTitleFromPaymentInstrument = (
  paymentInstrument: PaymentInstrument
) => `${FOUR_UNICODE_CIRCLES} ${paymentInstrument.panPartialNumber ?? ""}`;
```

in order to have:

![Schermata 2021-03-18 alle 12 10 41](https://user-images.githubusercontent.com/26501317/111617073-f6542380-87e2-11eb-9b06-831421973ae0.png)

Reply:
Sounds good, I updated also the `getTitleFromCard` function as:
```typescript
export const getTitleFromCard = (
  creditCard: RawCreditCardPaymentMethod | RawPrivativePaymentMethod
) => `${FOUR_UNICODE_CIRCLES} ${creditCard.info.blurredNumber ?? ""}`;
```
in [cf5c1c4](https://github.com/pagopa/io-app/pull/2916/commits/cf5c1c44f60dc1f68765cea1a52df0fdf5601010)